arkouda/client.py

```diff
@@ -129,7 +129,7 @@ def connect(
 
     Returns
     -------
-    None
```

Comment:
I don't see a `return` in this function, so I don't think it returns anything. I might be missing something